front/app/containers/ProjectsShowPage/timeline/VotingResults/index.tsx

```diff
@@ -6,22 +6,53 @@ import useIdeas from 'api/ideas/useIdeas';
 // components
 import { Box, useBreakpoint } from '@citizenlab/cl2-component-library';
 import VotingResultCard from './VotingResultCard';
+import { VotingMethod } from 'api/phases/types';
 
 interface Props {
   phaseId: string;
+  votingMethod: VotingMethod;
 }
 
-const VotingResults = ({ phaseId }: Props) => {
+const VotingResults = ({ phaseId, votingMethod }: Props) => {
   const { data: ideas } = useIdeas({
     phase: phaseId,
-    sort: 'votes_count',
+    sort: votingMethod === 'budgeting' ? 'baskets_count' : 'votes_count',
   });
-
   const smallerThanPhone = useBreakpoint('phone');
 
+  if (!ideas) return null;
+
   const getMx = (i: number) =>
     smallerThanPhone ? {} : i % 2 ? { ml: '8px' } : { mr: '8px' };
 
+  const getRanking = (
```

Comment:
We use this to show ties. Idea cards with the same ranking will have the same #. For example: 
<img width="1242" alt="Screenshot 2024-02-05 at 16 02 00" src="https://github.com/CitizenLabDotCo/citizenlab/assets/16427929/02077c55-9ee5-4c41-9c60-bf28554fa09e">